packages/sanity/src/core/store/_legacy/document/document-pair/remoteSnapshots.ts

```diff
@@ -17,9 +18,5 @@ export const remoteSnapshots = memoize(
       switchMap(({published, draft}) => merge(published.remoteSnapshot$, draft.remoteSnapshot$))
     )
   },
-  (client, idPair, typeName) => {
```

Comment:
Not removed! Just replaced with `memoizeKeyGen` which is also being used in a few places throughout the studio. I thought it'd make sense to reference that here too, though can easily revert if you think this would need to be keyed differently in future.

Reply:
Ah, missed that - thanks for clearing that up!